### PR TITLE
Fix immediate flush of existing connections that are stolen

### LINKED ISSUE #1870: tcp stealing doesn't take effect
### Bug Description

When there's a connection to the "mirrord" service with keepalive, on some Linux nodes the redirect rule doesn't apply and the conntrack flush doesn't make it take effect.
I manually added a rule
```
iptables -A INPUT -p tcp --dport <port> -j REJECT --reject-with tcp-reset
```
then after I saw the reset I removed it, then it worked.
Adding a rule for a moment doesn't help, since it needs to receive a packet then it'd reject it (if you add + remove and there wasn't a packet between the interval, the connection wasn't reset)

### Steps to Reproduce

not sure

### Backtrace

_No response_

### Relevant Logs

_No response_

### Your operating system and version

n/a

### Local process

n/a

### Local process version

_No response_

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by aviramha**
Closes #1870

### CODE REVIEW

File: mirrord/agent/src/steal/ip_tables/chain.rs [hunk 1]
```diff
@@ -59,6 +59,7 @@ where
         &self.inner
     }
 
+    #[tracing::instrument(skip(self))]
```
Comment (DmitryDodzin) [inline mirrord/agent/src/steal/ip_tables/chain.rs:62]: Add to all functions or remove =] and `level = "trace"`

**Review by DmitryDodzin — APPROVED** [review]
Nitpick

**Comment by aviramha** [comment]
Waiting with merge until user confirms it works (right now it doesn't)

File: mirrord/agent/src/steal/ip_tables/flush_connections.rs [hunk 2]
```diff
@@ -1,49 +1,106 @@
...
+            "-p tcp -m connmark --mark {MARK} -j REJECT --reject-with tcp-reset"
+        ))?;
+
+        Ok(FlushConnections { managed, inner })
+    }
+
+    #[tracing::instrument(level = "trace", skip(ipt, inner))]
+    pub fn load(ipt: Arc<IPT>, inner: Box<T>) -> Result<Self> {
+        let managed =
+            IPTableChain::create(ipt.with_table("filter").into(), IPTABLE_INPUT.to_string())?;
```
Comment (DmitryDodzin) [inline mirrord/agent/src/steal/ip_tables/flush_connections.rs:51]:
```suggestion
            IPTableChain::load(ipt.with_table("filter").into(), IPTABLE_INPUT.to_string())?;
```

**Pull request merged**